python/cuml/tests/dask/test_dask_logistic_regression.py

```diff
@@ -653,3 +658,349 @@ def test_exception_one_label(fit_intercept, client):
+    cpu = CPULR(
+        solver=sk_solver,
+        fit_intercept=fit_intercept,
+        penalty=penalty,
+        l1_ratio=l1_ratio,
+        C=C,
+    )
+    cpu.fit(X_train_scaled, y_train)
+    cpu_preds = cpu.predict(X_test_scaled)
+    cpu_accuracy = accuracy_score(y_test, cpu_preds)
+
+    assert len(mgon_preds) == len(cpu_preds)
+    assert (mgon_accuracy >= cpu_accuracy) | (
+        np.abs(mgon_accuracy - cpu_accuracy) < 1e-3
+    )
+
+    # assert equal the accuracy and the model
+    mgon_coef_origin = mgon.coef_.to_numpy() * scaler.scale_
+    if fit_intercept is True:
+        mgon_intercept_origin = mgon.intercept_.to_numpy() + np.dot(
+            mgon.coef_.to_numpy(), scaler.mean_
+        )
+    else:
+        mgon_intercept_origin = mgon.intercept_.to_numpy()
+
+    if sk_solver == "lbfgs":
+        assert array_equal(mgon_coef_origin, cpu.coef_, tolerance)
+        assert array_equal(mgon_intercept_origin, cpu.intercept_, tolerance)
+
+    # running MG with standardization=False
+    mgoff = cumlLBFGS_dask(
+        standardization=False,
+        solver="qn",
+        fit_intercept=fit_intercept,
+        penalty=penalty,
+        l1_ratio=l1_ratio,
+        C=C,
+        verbose=True,
+    )
+    X_train_ds, X_test_ds, y_train_dask, _ = to_dask_data(
+        X_train_scaled, X_test_scaled, y_train, y_test
+    )
+    mgoff.fit(X_train_ds, y_train_dask)
+    mgoff_preds = (
+        mgoff.predict(X_test_ds, delayed=delayed).compute().to_numpy()
+    )
+    mgoff_accuracy = accuracy_score(y_test, mgoff_preds)
+
+    # assert equal the accuracy and the model
+    assert len(mgon_preds) == len(mgoff_preds)
+    assert (mgon_accuracy >= mgoff_accuracy) | (
+        np.abs(mgon_accuracy - mgoff_accuracy) < 1e-3
+    )
+
+    print(f"mgon_coef_origin: {mgon_coef_origin}")
+    print(f"mgoff.coef_: {mgoff.coef_.to_numpy()}")
+    print(f"mgon_intercept_origin: {mgon_intercept_origin}")
+    print(f"mgoff.intercept_: {mgoff.intercept_.to_numpy()}")
+    assert array_equal(mgon_coef_origin, mgoff.coef_.to_numpy(), tolerance)
+    assert array_equal(
+        mgon_intercept_origin, mgoff.intercept_.to_numpy(), tolerance
+    )
+
+
+@pytest.mark.mg
+@pytest.mark.parametrize("fit_intercept", [True, False])
+@pytest.mark.parametrize(
+    "regularization",
+    [
+        ("none", 1.0, None),
+        ("l2", 2.0, None),
+        ("l1", 2.0, None),
+        ("elasticnet", 2.0, 0.2),
+    ],
+)
+def test_standardization_example(fit_intercept, regularization, client):
+    datatype = np.float32
+    n_parts = 2
+    max_iter = 5  # cannot set this too large. Observed GPU-specific coefficients when objective converges at 0.
+
+    penalty = regularization[0]
+    C = regularization[1]
+    l1_ratio = regularization[2]
+
+    est_params = {
+        "penalty": penalty,
+        "C": C,
+        "l1_ratio": l1_ratio,
+        "fit_intercept": fit_intercept,
+        "max_iter": max_iter,
+    }
+
+    X = np.array(
```

Comment:
Please see below for comments on hardcoding these values and generating larger test cases.

Reply:
Revised.